components/org.wso2.carbon.identity.policy.password/src/main/java/org/wso2/carbon/identity/policy/password/validation/PasswordValidationService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.identity.policy.password.validation;
+
+/**
+ * This is the api for password validation against the password policy oset by the Identity Server/System admin.
+ */
+public interface PasswordValidationService {
```

Comment:
A getter and setter for regex is already in the PasswordPolicyBean class
Do we need an api for get and change the regex?